src/components/Chat/BottomSheetAttachment.js

```diff
@@ -36,7 +36,7 @@ const BottomSheetAttachment = React.forwardRef((props, ref) => (
             <Text style={styles.textCard}>Send a GIF</Text>
           </View>
           {props.isLoadingUploadGIF ? (
-            <ActivityIndicator size="small" color={COLORS.white2} />
+            <ActivityIndicator size="small" color={COLORS.white} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the color of the `ActivityIndicator` has been changed from `COLORS.white2` to `COLORS.white`. Make sure this change doesn't affect the visibility and accessibility of the component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->